### PR TITLE
Loads HTML into the demo editor.

### PR DESCRIPTION
**Pull request by diegoreymendez**
**What works:**

Loads HTML into the demo editor.

**What doesn't work:**

Unfortunately there's an issue with how we've implemented the formatting that's making it not work properly.  In any case, since there's value in having the parser wired, I'm submitting this PR now and will work in parallel on a fix for the formatting issues.

**How to test:**

Just launch the demo app, go to "Editor Demo" and make sure the HTML text is loaded.  Some styles may be present, but they will mostly not work yet.

### CODE REVIEW

File: Classes/Private/NSAttributedString/Converters/HTMLNodeToNSAttributedString.swift [hunk 1]
```diff
@@ -133,7 +197,7 @@ class HMTLNodeToNSAttributedString: SafeConverter {
             string.addAttribute(NSStrikethroughStyleAttributeName, value: 1, range: fullRange)
         }
 
-        string.addAttribute(NSFontAttributeName, value: font(forNode: node), range: fullRange)
+        //string.addAttribute(NSFontAttributeName, value: font(forNode: node), range: fullRange)
```
Comment (aerych) [inline Classes/Private/NSAttributedString/Converters/HTMLNodeToNSAttributedString.swift]: still needed?

File: Classes/Public/Converters/HTMLToAttributedString.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    public func convert(html: String) throws -> NSAttributedString {
+        let htmlToNode = Libxml2.In.HTMLConverter()
+        let nodeToAttributedString = HMTLNodeToNSAttributedString(usingDefaultFontDescriptor: defaultFontDescriptor)
+
+        let node = try htmlToNode.convert(html)
+
+        return nodeToAttributedString.convert(node)
+    }
+}
```
Comment (aerych) [inline Classes/Public/Converters/HTMLToAttributedString.swift]: Maybe add a new line at the end of the file?

**Comment by aerych** [comment]
:shipit: from me :)

**Comment by diegoreymendez** [comment]
Ty @aerych.

**Pull request merged**